### DISCUSSION TITLE
Apply transform to a camera

### DISCUSSION THREAD

**Discussion by sergei9838** [marcomusy/vedo#1049] [Ideas]
Hi, Marco,
It would be nice and logical that the following code works:
```
from vedo import *
C = Cube()
cam = show(C).camera  # getting camera from the plot
close()

# define a transform:
C.shift((1,1,1)).rotate_x(30)
T = C.transform

# trying to apply it to the camera:
cam1 = cam.ApplyTransform(T)
# TypeError: ApplyTransform argument 1: method requires a VTK object

```
Visibly, vtkCamera wants a vtkTransform, but T is vedo.transformations.LinearTransform. Yes, I can create a new vtk camera by applying T (or T.matrix) to the focal point, position and view-up vector, 
but maybe there is an easier solution? For instance, transform T to vtkTransform and apply it to cam?

Or, in perspective, to make cam a new class vedo.camera with all the niceties that you already have for transforms of pointcloud?
Thank you!

**Comment by marcomusy** [comment]
You can access the `vtkTransform` object with `.T`, eg:

```py

from vedo import *

C = Cube()

plt = Plotter(axes=1)
plt.show(C, "press q")

# define a transform:
LT = LinearTransform()
LT.shift([1,0,0]).rotate_x(60)
print(LT)

# apply it to the camera:
plt.camera.ApplyTransform(LT.T)
plt.interactive()
```

**Reply by sergei9838** [reply]
Thank you, Marco!!!
